packages/forms/src/elements/file-list/utils.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import FilePdfStroke from '@zendeskgarden/svg-icons/src/16/file-pdf-stroke.svg';
+import FileZipStroke from '@zendeskgarden/svg-icons/src/16/file-zip-stroke.svg';
+import FileImageStroke from '@zendeskgarden/svg-icons/src/16/file-image-stroke.svg';
+import FileDocumentStroke from '@zendeskgarden/svg-icons/src/16/file-document-stroke.svg';
+import FileSpreadsheetStroke from '@zendeskgarden/svg-icons/src/16/file-spreadsheet-stroke.svg';
+import FilePresentationStroke from '@zendeskgarden/svg-icons/src/16/file-presentation-stroke.svg';
+
+export type FILE_TYPE = 'pdf' | 'zip' | 'image' | 'document' | 'spreadsheet' | 'presentation';
+
+export const ARRAY_FILE_TYPE: FILE_TYPE[] = [
+  'pdf',
+  'zip',
+  'image',
+  'document',
+  'spreadsheet',
+  'presentation'
+];
```

Comment:
Is it possible to use an enum here over repeating the strings twice? Fell free to ignore if this complicates stuff.

Reply:
Yeah it's possible. I've refactored this to use `enum` instead of `type`.  Looks like a `type` can be derived from an `enum` using `keyof typeof`. The `enum` is also flexible to be used as an object (albeit with a few caveats).

### Before

```typescript
export type FILE_TYPE = 'pdf' | 'zip' | 'image' | 'document' | 'spreadsheet' | 'presentation';

export const ARRAY_FILE_TYPE: FILE_TYPE[] = [
  'pdf',
  'zip',
  'image',
  'document',
  'spreadsheet',
  'presentation'
];
```

### After

```typescript
export enum FileType {
  pdf = 'pdf',
  zip = 'zip',
  image = 'image',
  document = 'document',
  spreadsheet = 'spreadsheet',
  presentation = 'presentation'
}

export type FILE_TYPE = keyof typeof FileType;

export const ARRAY_FILE_TYPE: FILE_TYPE[] = [...Object.values(FileType)];
```

https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-compile-time